packages/beacon-node/src/chain/seenCache/seenAttestationData.ts

```diff
@@ -11,8 +11,8 @@ type AttDataBase64 = string;
 export type AttestationDataCacheEntry = {
   // part of shuffling data, so this does not take memory
   committeeValidatorIndices: Uint32Array;
-  // TODO: remove this? this is available in SingleAttestation
```

Comment:
@twoeths I am not sure about this comment, seems simpler to keep the committee index in the cache as if in case of loading the attestation from cache we won't deserialize the `SingleAttestation` to get the committe index from there, and for pre-electra we would still need it.

Reply:
yes let's keep it in the cache and remove "TODO"